CONTRIBUTING.md

```diff
@@ -0,0 +1,99 @@
+# Contents
+* [Notes](#notes)
+* [Contributing](#contributing)
+* [Resolving Merge Conflicts](#resolving-merge-conflicts)
+* [Best Practices for Contributing](#best-practices-for-contributing)
+* [Attribution](#attribution)
+
+# Notes
+[Notes]: #notes
+
+Any contributions received are assumed to be covered by the BSD 3-Clause license. We might ask you to sign a Contributor License Agreement before accepting a larger contribution. To learn more about ReproZip, see:
+* [ReproZip Examples](https://vida-nyu.github.io/reprozip-examples/)
+* [ReproZip Documentation](https://reprozip.readthedocs.io/)
+* [ReproZip Demo Video](https://www.youtube.com/watch?v=-zLPuwCHXo0)
+
+# Contributing
+[Contributing]: #contributing
+
+Please note we use the [Citizen Code of Conduct](https://github.com/stumpsyn/policies/blob/master/citizen_code_of_conduct.md), please follow it in all your interactions with the project. If you would like to contribute to this project by modifying/adding to the code, please read the [Best Practices for Contributing] below and feel free to follow the standard Github workflow:
```

Comment:
- Instead of "please note ... please follow...", what about simply "please follow ..." ?
- Github -> GitHub